platform/machine/qemu/flight.go

```diff
@@ -124,7 +85,4 @@ func (qf *flight) NewCluster(rconf *platform.RuntimeConfig) (platform.Cluster, e
 
 func (qf *flight) Destroy() {
```

Comment:
I believe this method can be dropped and we'll inherit the embedded `LocalFlight`'s `Destroy` method.